packages/core/src/components/koros/koros.stories.js

```diff
@@ -20,6 +20,22 @@ export const Basic = () => `
   </div>
 `;
 
+export const Dense = () => `
```

Comment:
Should this be dense basic? Technically, we should have all the dense type examples, but I think we can go with only this for now.

Reply:
But aren't the Core Koros purely just SVGs? So the developer is supposed to just copy these? We need to provide all examples if this is the case.

Reply:
Yes. Then we probably should have all the dense variants.

Reply:
- [ ] Add dense examples of all types